src/app/components/form/input/FormInput.vue

```diff
@@ -111,13 +111,13 @@ const {
   isValidatable,
   idLabel,
   placeholder,
-  success,
+  // success,
```

Comment:
Comments like this can be removed when the usages in code (above here) are removed instead of commented out as well.